Dockerfile.mustgather

```diff
@@ -1,10 +1,9 @@
-FROM registry.ci.openshift.org/ocp/4.8:must-gather AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+WORKDIR /go/src/github.com/openshift/local-storage-operator/must-gather
+COPY . .
 
-FROM registry.ci.openshift.org/ocp/4.8:cli
-
-RUN microdnf -y install rsync
-
-COPY --from=builder /usr/bin/oc /usr/bin/oc
-COPY must-gather/* /usr/bin/
+FROM registry.svc.ci.openshift.org/ocp/4.7:must-gather
```

Comment:
Why are we using 4.7 templates?

Reply:
I updated this to be the 4.8 image. Note that I also updated the builder image to be based on 4.8 as well.